recipes/uncertainty-toolbox/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "uncertainty-toolbox" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 8c45695b1633e47ac743fa420eac05e6454299cf7ec9d06b2710d90c836eb4cb
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy >=1.19.0
+    - scipy >=1.5.0
+    - matplotlib-base >=3.2.2
+    - scikit-learn >=0.23.1
+    - shapely >=1.6.4.post2
+    - tqdm >=4.54.0
+    - pytest >=5.4.3
+    - black >=19.10b0
```

Comment:
I took the dependencies from https://github.com/uncertainty-toolbox/uncertainty-toolbox/blob/master/requirements/requirements.txt as that is what is also used in the `setup.py`.